A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js

```diff
@@ -93,7 +93,17 @@ class CDUInitPage {
             let value = mcdu.inOut;
             mcdu.clearUserInput();
             mcdu.tryUpdateFromTo(value, (result) => {
-                if (result) {
+                if (result) {                    
+                    let thrRedAccAltitude;
+                    
+                    const origin = mcdu.flightPlanManager.getOrigin();
+                    if (origin) {
+                        thrRedAccAltitude = origin.altitudeinFP + 1500;
+                    }
+                    
+                    mcdu.thrustReductionAltitude = thrRedAccAltitude;
```

Comment:
Wouldn't it be better to have

```
// let thrRedAccAltitude; // removed

const origin = mcdu.flightPlanManager.getOrigin();
if (origin && origin.altitudeinFP) {
    const thrRedAccAltitude = origin.altitudeinFP + 1500;
    mcdu.thrustReductionAltitude = thrRedAccAltitude;
    mcdu.accelerationAltitude = thrRedAccAltitude;
}
```

because otherwise "NULL" is set to `thrustReductionAltitude ` and `accelerationAltitude` if origin doesn't exist? Then leave the values as they were. Also make sure `altitudeinFP` exists.

Reply:
This behaviour is intentional. When the CDU doesn't know what the origin airport is, or what the altitude of that airport is, THR RED/ACC are supposed to be empty.

Source:
![image](https://user-images.githubusercontent.com/4014454/92994402-32d45900-f4fa-11ea-9f6a-f6eb2cf0ef6f.png)

Reply:
Thanks for the clarification. Then that part is good.

Would be good to check availability of "altitudeinFP".